src/providers/ogr/qgsogrfeatureiterator.h

```diff
@@ -33,7 +33,6 @@ class QgsOgrFeatureSource : public QgsAbstractFeatureSource
     virtual QgsFeatureIterator getFeatures( const QgsFeatureRequest &request ) override;
 
   protected:
```

Comment:
Nope, just a legacy from the times when we used to use "protected" by default :-)